plugins/node.d/amavis

```diff
@@ -31,36 +33,34 @@ plugin usually runs at.  To enable log reading it needs to run as a
 group or user that has read access, as shown above.
 
 By default the logtail program is started without any explicit path,
-but if it is not found in the system $PATH then you will need to specify
+but if it is not found in the system $PATH then you'll need to specify
 the full path for the program.
 
 =head2 EXAMPLE CONFIGURATION
 
 The following shows a typical configuration:
 
   [amavis]
-     env.amavislog     /var/log/mail/mail.info
-     env.logtail       /usr/bin/logtail
+     env.amavislog     /var/log/maillog
+     env.logtail       /usr/local/bin/logtail
+     env.amavisd       /usr/local/sbin/amavisd
```

Comment:
Same as above: please do not change a default, without a good reason (i.e. it being more or less universally applicable).